DBZ-4731 Adding note on MongoDB Atlas; (#3228)
debezium/debezium: documentation/modules/ROOT/pages/connectors/mongodb.adoc

You must also have a MongoDB user that has the appropriate roles to read the `admin` database where the oplog can be read. Additionally, the user must also be able to read the `config` database in the configuration server of a sharded cluster and must have `listDatabases` privilege action.
When change streams are used (the default) the user also must have cluster-wide privilege actions `find` and `changeStream`.

ifdef::community[]
[[mongodb-in-the-cloud]]
=== MongoDB in the Cloud

> The {prodname} connector for MongoDB can be used with https://www.mongodb.com/atlas/database[MongoDB Atlas].

Reviewer: ```suggestion
You can use the {prodname} connector for MongoDB with https://www.mongodb.com/atlas/database[MongoDB Atlas].
```

> Please make sure to have one of the xref:mongodb-property-capture-mode[`capture modes`] based on change streams (rather than oplog) enabled when connecting to MongoDB Atlas.

Reviewer: ```suggestion
When connecting {prodname} to MongoDB Atlas, enable one of the xref:mongodb-property-capture-mode[`capture modes`] to be based on change streams, rather than oplog.
```
Can you have multiple different capture modes in effect? It's not entirely clear to me whether this is a must have situation and whether we're saying that users should enable one of several capture modes to be based on change streams. If you can only use Atlas with change streams, I'd say something more like "When connecting {prodname} to MongoDB Atlas, set the xref:mongodb-property-capture-mode[`capture mode`] to use change streams rather than the oplog."

Author: We only have tested with change streams, hence the recommendation. Your proposal sounds good.
